src/post.ts

```diff
@@ -0,0 +1,23 @@
+import { info, warning } from '@actions/core'
+import {
+  getActionRef,
+  getMsSinceStart,
+  getOptionalEnvVariable,
+  getRequiredEnvVariable,
+  getRunUrl,
+  telemetryCollector,
+} from './util'
+
+if (getOptionalEnvVariable('LACEWORK_WROTE_TELEMERTY', 'false') !== 'true') {
```

Comment:
is this a typo?

Reply:
I think it's right. `getOptionalEnvVariable('LACEWORK_WROTE_TELEMERTY', 'false')` says take the value of `LACEWORK_WROTE_TELEMERTY` and if it's missing treat it as `false`. Then we check if that's `!== 'true'` to see if we didn't report telemetry.

Reply:
Oh, right, yes! It worked because I made the same typo in the other place haha.